app/services/scene-collections/nodes/root.ts

```diff
@@ -50,21 +53,24 @@ interface ISchema {
  * This is the root node of the config file
  */
 export class RootNode extends Node<ISchema, {}> {
-  schemaVersion = 4;
+  schemaVersion = 5;
```

Comment:
we don't usually need to update the schema if we're not depending on it's version or don't require a migration

Reply:
Reverted